tailscale/client.go

```diff
@@ -21,10 +21,11 @@ import (
 type (
 	// Client type is used to perform actions against the Tailscale API.
 	Client struct {
-		apiKey  string
-		http    *http.Client
-		baseURL *url.URL
-		tailnet string
+		apiKey    string
+		http      *http.Client
+		baseURL   *url.URL
+		tailnet   string
+		userAgent string
```

Comment:
// empty string means default?

Reply:
Done. Passing an empty string to use runtime's default seems like an odd API, but I am happy to leave a way to do that.